.github/workflows/publish-bugfix-release.yml

```diff
@@ -62,6 +62,8 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
this shouldn't be necessary unless there's the need to checkout to an existent commit (like it's done in the `helm-release` job)